cli/init/near-contracts.js

```diff
@@ -101,7 +101,7 @@ class InitNearContracts {
       hashesGcThreshold,
       finalizedGcThreshold,
       numConfirmations,
-      nearClientValidateEthhash === 'true',
+      nearClientValidateEthash === 'true',
```

Comment:
We should add code to make it fail if the value of `nearClientValidateEthash` is neither `'true'` nor `'false'`, to prevent errors (such as misspellings).